### ISSUE TITLE
Hand or foot lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9912]
**Preferred term label:**
Hand or foot lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Clavus durum

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

this is very general -- what is the use case?

**Comment by pnrobinson** [comment]
So these are types of corn?
https://www.ncbi.nlm.nih.gov/books/NBK470374/
see #9911

**Comment by MickeySegal** [comment]
I changed our finding name to "Hand or foot localization to lesions"

**Comment by pnrobinson** [comment]
Would 
 HP:0012839
Distal
do?

**Comment by MickeySegal** [comment]
It is the same idea but more specifically as palmoplantar.  I note that HPO has palmoplantar applied to various types of lesions.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001109
- label: Palmoplantar location
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applied to lesions whose distribution is confined or restricted to the palms of the hands and soles of the feet.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9912
- synonyms:
